forest/daemon/src/lib.rs

```diff
@@ -1,4 +1,30 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
+
 pub mod bundle;
-pub mod cli;
+pub mod daemon;
+
+use raw_sync::events::{Event, EventInit as _};
+use shared_memory::ShmemConf;
+
+use lazy_static::lazy_static;
+use tempfile::{Builder, TempPath};
+
+lazy_static! {
+    static ref IPC_PATH: TempPath = Builder::new()
```

Comment:
nit: Perhaps it's time to replace `lazy_static!` with `once_cell::sync::Lazy`. It should be even easier once [this](https://github.com/ChainSafe/forest/pull/2970) is merged, then all it takes is `std::sync::Lazy`.

Reply:
Yep, you'll also be glad to learn that we have a lot of `Lazy<Vec<String>>` which should be `const ...: &[&str]`.

Once our code is cleaner I'll move on to refactoring more stuff

Reply:
#2972